internal/coreinternal/processor/filterconfig/config.go

```diff
@@ -128,8 +129,12 @@ func (mp *MatchProperties) ValidateForLogs() error {
 		return errors.New("neither services nor span_names should be specified for log records")
 	}
 
-	if len(mp.LogNames) == 0 && len(mp.Attributes) == 0 && len(mp.Libraries) == 0 && len(mp.Resources) == 0 {
-		return errors.New(`at least one of "log_names", "attributes", "libraries" or "resources" field must be specified`)
+	if len(mp.LogNames) > 0 {
+		return errors.New("log_names setting is deprecated and will be removed soon")
```

Comment:
This is not what I intended to do. I think this needs to be a warning. Don't merge.

Reply:
The logger is not available here. It used to be available in component creation but I cannot see it anymore. Did we break it?